easybuild/easyconfigs/h/HEALPix/HEALPix-3.50-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,25 @@
+name = 'HEALPix'
+version = '3.50'
+
+homepage = 'http://healpix.sourceforge.net/'
+description = """Hierarchical Equal Area isoLatitude Pixelation of a sphere."""
+
+# basic_gcc, generic_gcc, optimized_gcc targets available
+gcc_target = 'optimized_gcc'
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
```

Comment:
@jose-d Using `GCC/7.3.0-2.30` as a toolchain would make more sense here, since that leaves the door open to also have an easyconfig using `iccifort` as toolchain.

Since `GCCcore` sits underneath both `GCC` and `iccifort`, having both `GCCcore` and `iccifort` should be avoided.

Reply:
mhmm I see your point and it would make perfect sense for some advanced/computing-demanding software.. 

But HEALPix is* used as kind of a intput/output "plugin" to pixelize sphere. The real compute-demanding part is not done in Healpix, but by the software around..

The last healpix-app I was working ended being builded with `pgc++` and having the compute-heavy parts at the GPU.. The part done by healpix was like small units of % of the total time..

So the benefit having GCCcore vesion of Healpix I can glue to (gnu|intel|pgi) stack was higher than having less time spent in healpix.

But if you know(?) apps spending lot of time in healpix calls, then it's off-course good motivation to make it better (=to use more fancy compiler).

* at my $institute by my $users

Reply:
@jose-d I was assuming that HEALPix was compute-heavy since it has support for building with Intel compilers.

But if it's not compute-heave at all, then there's no benefit in building with Intel compilers, indeed, and we can leave it as is and use the `GCCcore` toolchain.

Thanks a lot for clarifying!